src/uploader.js

```diff
@@ -50,22 +52,31 @@ define([
                 'description': 'Video',
                 'extensions': '*.3gp;*.mp4'
             }
-        ]
+        ],
+        text: [
+            {
+                'description': 'HTML',
```

Comment:
This does seem a little wrong, but the current plan is to use only HTML files, but in the hope of keeping the HQ side a little simpler I just made a generic text upload.